content/applications/hr/appraisals/reporting.rst

```diff
@@ -0,0 +1,188 @@
+=========
+Reporting
+=========
+
+In Odoo's *Appraisals* app, two metrics are tracked as appraisals are completed: an :ref:`appraisal
+analysis <appraisals/analysis>`, and a :ref:`skills evolution <appraisals/skills-report>`.
+
+.. _appraisals/analysis:
+
+Appraisal analysis
+------------------
+
+To access the *Appraisal Analysis* report, navigate to :menuselection:`Appraisals app --> Reporting
+--> Appraisal Analysis`.
+
+On the :guilabel:`Appraisal Analysis` page, there is a report of all the appraisals in the database,
+highlighted in different colors to represent their status.
+
+Appraisals in yellow are completed, appraisals in orange are in progress (the appraisal is
+confirmed, but not completed), appraisals in red have been cancelled, and appraisals in gray are
+scheduled (according to the :ref:`appraisals/appraisal-plan`), but have not been confirmed yet.
+
+The report displays the current year, in a default Gantt view, and is grouped by department.
+
+To change the period of time that is presented by default, adjust the date settings in the top-left
+of the report. The options to display are :guilabel:`Day`, :guilabel:`Week`, :guilabel:`Month`, and
+:guilabel:`Year`. Use the arrows to move forward or backward in time.
+
+At any point, click the :guilabel:`Today` button to have the Gantt view include today's date in the
+view.
+
+The report can have other filters and groupings set in the :guilabel:`Search...` bar at the top.
```

Comment:
Since I am suggesting to remove the essentials based Filter appraisals section, can instead link directly to the essentials content
```suggestion
The report can have other :ref:`filters <search/filters>` and :ref:`groupings <search/group>` set in
the :guilabel:`Search...` bar at the top.
```